docs/source/llms/tracing/index.rst

```diff
@@ -227,6 +232,70 @@ for model/API invocations to the active MLflow Experiment.
             :width: 100%
             :align: center
 
+    .. tab:: Swarm
+
+        .. raw:: html
+
+            <h3>OpenAI Swarm Automatic Tracing</h3>
+
+        |
```

Comment:
```suggestion
```

do we need this vertical line?

Reply:
Good question, this is for adding some margin between title and paragraph.

<img width="400" alt="Screenshot 2024-10-22 at 20 47 20" src="https://github.com/user-attachments/assets/a2ae7b19-3b22-4269-978b-f3bb8a5669a4">
<img width="400" alt="Screenshot 2024-10-22 at 20 46 52" src="https://github.com/user-attachments/assets/e111247d-f593-44a0-a0ef-bf4f29525556">

I thought it looks nice when I first added these tabs, but now I don't feel there is much difference lol. Do you think we should keep it or remove?

Reply:
I think we should keep it. Having the clear whitespace, particularly for some follow-on header level statements, looks cleaner.